blocks/columns/columns.css

```diff
@@ -19,15 +19,62 @@
   display: block;
 }
 
-@media (min-width: 900px) {
+@media (min-width: 1023px) {
   .columns > div {
     align-items: center;
     flex-direction: unset;
-    gap: 32px;
   }
 
   .columns > div > div {
     flex: 1;
     order: unset;
   }
+
+  .pod > div > div:not(.columns-img-col) * {
+    padding-left: 70px;
+    padding-right: 70px;
+    max-width: 405px;
+  }
+}
+
+.pod h2 {
```

Comment:
it's generally a good idea to put media queries at the end, to make defaults easier to overwrite

Reply:
👍